internal/utils/utils.go

```diff
@@ -13,3 +17,80 @@ func HasFinalizer(cr *metav1.ObjectMeta, name string) bool {
 	}
 	return false
 }
+
+func ExtractInfoFromEnvFrom(envs []corev1.EnvFromSource, themap MapType, ref func(interface{}) (bool, *bool, string)) {
```

Comment:
WDYT about aliasing `func(interface{}) (bool, *bool, string)` so it was clear what it is supposed to do?